res/linuxPackaging.json

```diff
@@ -18,8 +18,8 @@
     "x-scheme-handler/ftp"
   ],
   "requires": [
-    "GConf2",
-    "libXScrnSaver",
+    "(GConf2 or gconf2)",
+    "(libXScrnSaver or libXss1)",
```

Comment:
@Garbee do you mean this PR **breaks** the installation on the current stable OpenSUSE version, or the installation has never worked on OpenSUSE at all?

If former, we should not merge the PR.
If latter, I think the PR is still acceptable.
If this PR does not fix anything, it should be closed.

Do you have a better solution for the leap? If so, would you mind letting us know what it is like? thanks!